linux/shim/service.go

```diff
@@ -46,7 +46,10 @@ func NewService(path, namespace, address string) (*Service, error) {
 		client = events.NewEventsClient(conn)
 	} else {
 		client = &localEventsClient{
-			emitter: evt.GetPoster(context),
+			// TODO(stevvooe): This doesn't look entirely correct. Where do
+			// these events go? Looks like a bug, as we were pulling nil poster
+			// out of context.
+			forwarder: evt.NewExchange(),
```

Comment:
Where does this need to be instantiated? I may be confused on the intended behavior. What will consume from this exchange?